src/main/java/org/web3j/console/Runner.java

```diff
@@ -43,6 +45,18 @@
     public static void main(String[] args) throws Exception {
         System.out.println(LOGO);
 
+        CliConfig config = CliConfig.getConfig();
+        Updater updater = new Updater(config);
+        if (!updater.updateCurrentlyAvailable()) {
+            new Thread(updater::onlineUpdateCheck).start();
```

Comment:
Maybe we should run this even if updater.updateCurrentlyAvailable() returns true?

Reply:
Fixed this